go.mod

```diff
@@ -21,16 +21,16 @@ require (
 	golang.org/x/sync v0.3.0
 	golang.org/x/text v0.13.0
 	golang.org/x/time v0.3.0
-	k8s.io/api v0.26.6
-	k8s.io/apiextensions-apiserver v0.26.6
-	k8s.io/apimachinery v0.26.6
-	k8s.io/client-go v0.26.6
+	k8s.io/api v0.28.2
+	k8s.io/apiextensions-apiserver v0.28.0
```

Comment:
Why is this at a different version? Typically all of the k8s client package versions are able to match

Reply:
ya done